packages/core/core-flows/src/product/workflows/update-products.ts

```diff
@@ -148,7 +212,7 @@ export const updateProductsWorkflow = createWorkflow(
   (
     input: WorkflowData<WorkflowInput>
   ): WorkflowData<ProductTypes.ProductDTO[]> => {
-    // TODO: Delete price sets for removed variants
+    const previousVariantIds = getVariantIdsForProductsStep(input)
```

Comment:
sugg: you can use `useRemoteQueryStep` directly here and a transform to get the ids

Reply:
+1, if possible

Reply:
I'll just have this merged as-is so it doesn't linger for too long, but if you think we should still go with useRemoteQueryStep I'll just handle it in a separate PR